### PR TITLE
Update and Delete bits

### LINKED ISSUE #27: Add ability to delete bits

### LINKED ISSUE #28: Add ability to edit bits

### PR DESCRIPTION
**Pull request by edwardloveall**
Closes #27
Closes #28

### CODE REVIEW

File: src/actions/bits/edit.cr [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Bits::Edit < BrowserAction
+  route do
+    bit = BitQuery.find(params.get(:bit_id))
```
Comment (paulcsmith) [inline src/actions/bits/edit.cr]: You can use “bit_id” so you get type safe params  :)

File: spec/flows/bit_spec.cr [hunk 2]
```diff
@@ -12,4 +12,32 @@ describe "User visits bit homepage" do
...
+
+  it "edits one of their bits" do
+    user = UserBox.create
+    flow = BitFlow.new(user: user)
+    bit = BitBox.create(&.user_id(user.id))
+
+    flow.sign_in
+    flow.visit_my_page
+    flow.click_on_edit_bit(bit)
+    flow.el("[name='bit:title']").clear
```
Comment (paulcsmith) [inline spec/flows/bit_spec.cr]: Is this because of a bug in LuckyFlow?

Comment (edwardloveall) [reply spec/flows/bit_spec.cr]: Hah! I was literally filling out a LuckyFlow issue. It's not a bug, exactly. The short of it that when it fills out fields, it only appends. I'd get fields like `Name: EdwardPaul`.

File: src/actions/bits/edit.cr (hunk 1 above)
Comment (edwardloveall) [reply src/actions/bits/edit.cr]: Ah thanks! I forgot you added plain methods to access params.

File: spec/support/flows/authenticated_base_flow.cr [hunk 3]
```diff
@@ -18,10 +18,18 @@ abstract class AuthenticatedBaseFlow < BaseFlow
...
+    visit_my_page
     sign_out_button.click
   end
 
+  def visit_my_page
+    visit Me::Show
+  end
+
+  def visit_bit_index_page
+    click "@main-follow-nav"
```
Comment (edwardloveall) [inline spec/support/flows/authenticated_base_flow.cr]: This is plain wrong

**Pull request merged**